_episodes/02-markdown.md

```diff
@@ -296,34 +259,31 @@ You can define either
 
 
 > ## Links
-> Fill in the blanks below to create the following rendered view:
->
-> ---
->
-> Examples:
-> 1. [Inline link](https://carpentries.org/)
-> 2. [Reference link][case-InSeNsiTiVe-reference-tag]
->
-> [case-insensitive-reference-tag]: https://carpentries.org/
+> Fill in the blanks in this code
 >
 > ~~~
-> Examples:
+> Example links:
 > 1. [___](https://carpentries.org/)
 > 2. [___]___case-InSeNsiTiVe-reference-tag___
 >
 > [case-insensitive-reference-tag]: https://carpentries.org/
 > ~~~
 > {: .source }
 >
+> to create the following rendered view:
+>
+> ![Preview tab of code blocks with syntax highlighting](../fig/exercise_links.png)
```

Comment:
Please update the alt text here. At the moment it describes the image (of rendered links) as showing code blocks with syntax highlighting.

Reply:
Thank you, see 63ecec4 for a fix of this copy paste mistake.